reinforcement_learning/README.md

```diff
@@ -21,6 +21,7 @@ These examples demonstrate how to train reinforcement learning models on SageMak
 -  [Roboschool Stable Baseline](rl_roboschool_stable_baselines) is an example of using [stable-baselines](https://stable-baselines.readthedocs.io/en/master/) to train RL algorithms.
 -  [Tic-tac-toe](rl_tic_tac_toe_coach_customEnv) uses RL to train a policy and then plays locally and interactively within the notebook.
 -  [Traveling Salesman and Vehicle Routing](rl_traveling_salesman_vehicle_routing_coach) is an example of using RL to address operations research problems.
+-  [Game Server Auto-pilot](rl_game_server_autopilot) Reduce player wait time by autoscale game-servers deployed in EKS cluster via RL approach by adding and removing EC2 instances in reaction to dynamically player usage.
```

Comment:
Reduce player wait time by **autoscaling** game-servers deployed in EKS cluster **using RL to add and remove** EC2 instances as per **dynamic** player usage.

Grammar fixes